drgn_tools/task.py

```diff
@@ -185,6 +191,133 @@ def get_command(task: Object) -> str:
     return escape_ascii_string(task.comm.string_())
 
 
+def get_ppid(task: Object) -> int:
+    """
+    :returns: Parent PID of the task
+    """
+    return task.parent.pid.value_()
+
+
+class TaskRss(NamedTuple):
+    """
+    : Represent's a task's resident set size. See task_rss().
```

Comment:
No need to have the colon at the beginning of this line.

Also it would be nice to mention that all the values are in units of pages. I clearly forgot to write that down :)